src/components/ReportActionItem/RenameAction.js

```diff
@@ -5,16 +5,22 @@ import Text from '../Text';
 import styles from '../../styles/styles';
 import reportActionPropTypes from '../../pages/home/report/reportActionPropTypes';
 import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import compose from '../../libs/compose';
+import withCurrentUserPersonalDetails, {withCurrentUserPersonalDetailsPropTypes} from '../withCurrentUserPersonalDetails';
 
 const propTypes = {
     /** All the data of the action */
     action: PropTypes.shape(reportActionPropTypes).isRequired,
 
     ...withLocalizePropTypes,
+    ...withCurrentUserPersonalDetailsPropTypes,
 };
 
 const RenameAction = (props) => {
-    const displayName = lodashGet(props.action, ['message', 0, 'text']);
+    const currentUserAccountID = lodashGet(props.currentUserPersonalDetails, 'accountID', '');
+    const userDisplayName = lodashGet(props.action, ['person', 0, 'text']);
+    const actorAccountID = lodashGet(props.action, 'actorAccountID', '');
+    const displayName = actorAccountID === currentUserAccountID ? `${props.translate('common.you')}` : `${userDisplayName}`;
```

Comment:
Why did we need to make this change to set "You" instead of the user's display name? It seems unnecessary.

Reply:
@neil-marcellini 
This was not part of the issue which we are fixing in LHN, this was a related translation issue where 'you' text was not translated in the chat page for the rename action. @Ollyws mentioned this issue and lended to fix it as well.

Since it was a similar translation issue during room rename action, I was ok with the extra fix as well. Let me know if you think otherwise.

https://github.com/Expensify/App/pull/20479#issuecomment-1583211684

Reply:
Ah ok I missed that. Thanks for explaining. Please add tests for that part as well.